packages/mobile/src/components/details-tile/MetadataRow.tsx

```diff
@@ -0,0 +1,18 @@
+import type { PropsWithChildren } from 'react'
+
+import { Flex, Text } from '@audius/harmony-native'
+
+type MetadataRowProps = PropsWithChildren<{
+  label: string
+}>
+
+export const MetadataRow = ({ label, children }: MetadataRowProps) => {
```

Comment:
this isn't exactly a row, its more a column, but maybe just drop the "row", and use "item" or something like that?